app/javascript/dashboard/routes/dashboard/settings/SettingsLayout.vue

```diff
@@ -0,0 +1,23 @@
+<script setup></script>
+<template>
+  <div class="flex flex-col w-full h-full gap-10 settings-page">
+    <slot name="header" />
+    <slot name="body" />
+  </div>
+</template>
+<style lang="scss" scoped>
+// To be removed once we have a Inter as a global font
+@import 'shared/assets/fonts/inter';
+.settings-page {
+  font-family:
+    Inter,
+    -apple-system,
+    system-ui,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    Roboto,
+    'Helvetica Neue',
+    Arial,
+    sans-serif !important;
+}
+</style>
```

Comment:
You can add this to the tailwind config, that would be better. Or else, we can start using Inter everywhere? @pranavrajs what do you think?

Also @iamsivin see if we can use [fontsource.org](https://fontsource.org) for this... perhaps, we could use it. That will ensure we can update the font easily